tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_component.scss

```diff
@@ -34,5 +34,5 @@ limitations under the License.
 }
 
 .horizontalFob {
```

Comment:
the css name should be dash-case instead of CamelCase

Reply:
Oh yea definitely. I changed the case.